SPECS/xapi.spec

```diff
@@ -622,6 +632,11 @@ mkdir -p %{buildroot}%{_sysconfdir}/xapi.pool-recommendations.d
 %py_byte_compile %{_pythonpath} %{buildroot}/%{_usr}/libexec/xenopsd
 %py_byte_compile %{_pythonpath} %{buildroot}/%{_sysconfdir}/xapi.d/plugins
 
+# For xs9, move this config here from xenserver-release
```

Comment:
same here. As it is related to xs9 should we keep this?

Reply:
Yes, there are other things related to XS 9 that we keep to avoid adding unneeded distance with XenServer's spec files. The difference is that they use the same spec files to build in both XS8 and XS9, and we only use them to build in XCP-ng 8.3. When comes the time to build for XCP-ng 9, we'll have to review all spec files anyway.